packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -192,6 +216,86 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     }
   }
 
+  const memberships = await prisma.membership.findMany({
+    where: {
+      userId: ctx.user.id,
+    },
+  });
+
+  const teamIds = memberships.map((membership) => membership.teamId);
+
+  // Send webhook to notify other services
+  const subscriberOptions: GetSubscriberOptions = {
+    userId: ctx.user.id,
+    teamIds: teamIds,
+    orgId: ctx.user.organizationId,
+    triggerEvent: WebhookTriggerEvents.OOO_CREATED,
+  };
+
+  const subscribers = await getWebhooks(subscriberOptions);
+
+  await Promise.all(
+    subscribers.map(async (subscriber) => {
+      sendPayload(
+        subscriber.secret,
+        WebhookTriggerEvents.OOO_CREATED,
+        dayjs().toISOString(),
+        {
+          appId: subscriber.appId,
+          subscriberUrl: subscriber.subscriberUrl,
+          payloadTemplate: subscriber.payloadTemplate,
+        },
+        {
+          oooEntry: {
+            id: createdRedirect.id,
+            start: dayjs(createdRedirect.start).tz(ctx.user.timeZone, true).format("YYYY-MM-DDTHH:mm:ssZ"),
+            end: dayjs(createdRedirect.end).tz(ctx.user.timeZone, true).format("YYYY-MM-DDTHH:mm:ssZ"),
+            createdAt: createdRedirect.createdAt.toISOString(),
+            updatedAt: createdRedirect.updatedAt.toISOString(),
+            notes: createdRedirect.notes,
+            reason: {
+              emoji: reason?.emoji,
+              reason: reason?.reason,
+            },
+            reasonId: input.reasonId,
+            user: {
+              id: ctx.user.id,
+              name: ctx.user.name,
+              username: ctx.user.username,
+              timeZone: ctx.user.timeZone,
+            },
+            toUser: toUserId
+              ? {
+                  id: toUserId,
+                  name: toUser?.name,
+                  username: toUser?.username,
+                  timeZone: toUser?.timeZone,
+                }
+              : null,
+            uuid: createdRedirect.uuid,
+          },
+          // Dummy data
+          attendees: [],
```

Comment:
I really don't like sending all the dummy data. We should only send the OOO data